app/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,29 @@
+import androidx.compose.desktop.ui.tooling.preview.Preview
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.WindowPosition
+import androidx.compose.ui.window.application
+import androidx.compose.ui.window.rememberWindowState
+import java.awt.Dimension
+import view.utils.SelectInitDialogWindow
+
+@Composable
+@Preview
+fun App() {
+    MaterialTheme {
+        SelectInitDialogWindow(true).GraphInitDialogWindow(true)
+    }
+}
+
+fun main() = application {
```

Comment:
> * When the window is resized the GUI gets stretched for a few seconds before coming back to norma

Hm, strange, i haven't found that)

>* It would be nice to have bigger graph as examples, a few hundred nodes at least. When I was doing this task 3 years ago we had a requirement to support full graphs with up to 10000 nodes (you don't have this requirement)

You are totally right!

>*  When creating a vertex I need to input some data to be stored in it but there seems to be no way to view that data anywhere in the app

Yep, only ID's. It would be good to implement (f.e. user needs to see what data have nodes inside the community))

>* The algorithm for finding SCCs crashes the app with java.lang.ArrayIndexOutOfBoundsException: Only 15 colors supported on "test_graph"

Seems to be and UI integration issue, cant say anything about it. SCC was totally tested as a unit

>* For the layout algorithm, the buttons don't seem to work, nothing happens when I press them. 💡 And what do each of the sliders represent? Some meaningful names would really help.

I repied to this issue in the comments below. Sliders represent the parameters of the T-FDP model, there is an explanation of it in the code. #toaddfaqbutton ;)